extract

```diff
@@ -45,6 +45,7 @@ parser.add_argument('end_year_month', nargs='?')
 
 parser.add_argument('-v', '--verbose', action='store_true')
 parser.add_argument('-o', '--output-dir', default='output')
+parser.add_argument('--accuracy-limit', type=int, default=None)
```

Comment:
When someone says "limit", I tend to think of a maximum, like "speed limit". In this case, the flag is actually specifying a minimum. Would something like `--min-accuracy` be more concise?

Reply:
Does 64d2ab0 help?

Reply:
Ah, I see now. Yes, that makes sense.